app/App/Account/Account/Balances/BalancesExpanded/index.js

```diff
@@ -60,9 +60,10 @@ class BalancesExpanded extends React.Component {
         const rate = isNative ? nativeCurrencyInfo : rates[rawBalance.address || rawBalance.symbol] || {}
         const logoURI = isNative ? nativeCurrencyInfo.icon : rawBalance.logoURI
         const name = isNative ? nativeCurrencyInfo.name || networks[rawBalance.chainId].name : rawBalance.name
-        const decimals = isNative ? 18 : rawBalance.decimals
+        const decimals = isNative ? nativeCurrencyInfo.decimals || 18 : rawBalance.decimals
+        const symbol = isNative ? nativeCurrencyInfo.symbol || rawBalance.symbol : rawBalance.symbol
```

Comment:
nice catch! you can simplify this a bit (I think) with:
`const symbol = (isNative && nativeCurrencyInfo.symbol) || rawBalance.symbol`